install/helm-repo/argocd-agent-agent/values.yaml

```diff
@@ -0,0 +1,22 @@
+# Default values for argocd-agent-agent.
```

Comment:
In general, it would make sense to have all settings available in the argocd-agent-params ConfigMap available as a Helm value too.

Reply:
Updated with missing keys from configMap agent-param-cm.